project_3.html

```diff
@@ -0,0 +1,407 @@
+					<li>Examples:</li>
+						<ul>
+							<li><code class="pj-code language-arduino">isEven(62) : true</code></li>
+							<li><code class="pj-code language-arduino">isEven(3) : false</code></li>
+							<li><code class="pj-code language-arduino">isEven(-2) : true</code></li>
+							<li><code class="pj-code language-arduino">isEven(0) : true</code></li>
+						</ul>
+				</ul>
+			</p>
+
+			<h4 class="mb-3 fw-light" id="pj-heading-number-power">Number Power</h4>
+			<p>
+				<ul class="mb-5">
+					<li>Function Header: <code class="pj-code language-arduino">unsigned pow2(unsigned base, unsigned exp)</code></li>
+					<li>Parameters: base (unsigned int), exp (unsigned int)</li>
+					<li>Return: unsigned int</li>
+					<li>Description: <code class="pj-code language-arduino">pow(2)</code> returns the calculated value of <code class="pj-code language-arduino">base</code> raised to the power of <code class="pj-code language-arduino">exp</code> (i.e. base <sup>exp</sup>). The parameters can only be positive integers.</li>
+					<li>Examples:</li>
+						<ul>
+							<li><code class="pj-code language-arduino">pow2(6, 2) : 36</code></li>
+							<li><code class="pj-code language-arduino">pow2(1, 8) : 1</code></li>
+							<li><code class="pj-code language-arduino">pow2(9, 3) : 729</code></li>
+						</ul>
+				</ul>
+			</p>
+
+			<h4 class="mb-3 fw-light" id="pj-heading-take-the-average">Take the Average</h4>
+			<p>
+				<ul class="mb-5">
+					<li>Function Header: <code class="pj-code language-arduino">double avg(int arr[], int size)</code></li>
+					<li>Parameters: arr (int array), size (int)</li>
+					<li>Return: double</li>
+					<li>Description: <code class="pj-code language-arduino">avg()</code> returns the mean of the elements in the array <code class="pj-code language-arduino">arr</code>. The array must not be mutated by the function. Its elements may be positive or negative.</li>
+				</ul>
+			</p>
+
+			<h4 class="mb-3 fw-light" id="pj-heading-fibonacci-sequence">Fibonacci Sequence</h4>
+			<p>
+				<ul class="mb-5">
+					<li>Function Header: <code class="pj-code language-arduino">void fib(unsigned n)</code></li>
+					<li>Parameters: n (unsigned int)</li>
+					<li>Return: None</li>
+					<li>Description: <code class="pj-code language-arduino">fib()</code> prints the first n elements of the Fibonacci sequence to the console. Each element should be separated by a comma and one space. The last element terminates the line without a space or comma. <code class="pj-code language-arduino">n > 0</code></li>
+					<li>Examples:</li>
+						<ul>
+							<li><code class="pj-code language-arduino">fib(1) : 1</code></li>
+							<li><code class="pj-code language-arduino">fib(4) : 1, 1, 2, 3</code></li>
+						</ul>
+				</ul>
+			</p>
+
+
+			<h2 class="mb-3 fw-bold" id="pj-heading-instructions">Instructions</h2>
+			<h4 class="mb-3" id="pj-heading-checkpoint1">Checkpoint 1: Hello World</h4>
```

Comment:
I think this would be a good place to insert an alert-warning container (similar to the ones we include in checkpoints) explaining the AI bot.

We could say something like this. "Replit comes equipped with AI assistive features, which we will not use for this project. In the IDE, expand the "Tools" dropdown in the left pane, and select "User settings." Then, a settings menu should appear. Inside the menu, turn OFF the "AI code completion" toggle. That's it!